packages/breadcrumbs/src/styled/StyledBreadcrumbItem.ts

```diff
@@ -6,23 +6,44 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
+import stripUnit from 'polished/lib/helpers/stripUnit';
+import { getColor } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.item';
 
 export interface IStyledBreadcrumbItemProps {
   isCurrent?: boolean;
 }
 
-export const StyledBreadcrumbItem = styled.li.attrs<IStyledBreadcrumbItemProps>(props => ({
+/**
+ * 1. CSS targets Microsoft Edge
+ */
+const anyLinkStyles = (styles: string) => `
+  & a {
+    &:any-link { ${styles} }
+    &:any-link:hover { ${styles} }
+    &:any-link:focus { ${styles} }
+    &:link { ${styles} } /* [1] */
+    &:visited { ${styles} } /* [1] */
+    &:focus:link { ${styles} } /* [1] */
+    &:hover:link { ${styles} } /* [1] */
+    &:focus:visited { ${styles} } /* [1] */
+    &:hover:visited { ${styles} } /* [1] */
+  }
+`;
+
+export const StyledBreadcrumbItem = styled.li.attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb__item'], {
-    // State
-    [BreadcrumbStyles['is-current']]: props.isCurrent
-  })
-}))<IStyledBreadcrumbItemProps>`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledBreadcrumbItemProps>`
+  ${props => `
+    display: flex;
+    align-items: center;
+    line-height: ${(props.theme.space.base * 5) / stripUnit(props.theme.fontSizes.md)};
+    white-space: nowrap;
+  `}
+  ${props =>
+    `color: ${props.isCurrent ? getColor(props.theme.colors.neutralHue, 600) : 'inherit'};`}
+  ${props => props.isCurrent && anyLinkStyles(`color: inherit;`)}
```

Comment:
If this is only used in 1 place, is there a reason to pass the color style as an attribute? This could be more performant if you include this value inline above.